use-cases/demos/Makefile

```diff
@@ -43,5 +47,14 @@ run_object_detection: gen_object_detection
 down_object_detection:
 	@PIPELINE_COUNT=$(PIPELINE_COUNT) RETAIL_USE_CASE_ROOT="$(RETAIL_USE_CASE_ROOT)" RESULTS_DIR="$(RESULTS_DIR)" docker compose -f add_demo_object_detection.yml down
 
-compose_down_all: down_classification down_instance_segmentation down_object_detection
+gen_pytorch_object_detection:
+	@PIPELINE_COUNT=$(PIPELINE_COUNT) RETAIL_USE_CASE_ROOT="$(RETAIL_USE_CASE_ROOT)" RESULTS_DIR="$(RESULTS_DIR)" docker compose -f add_demo_pytorch_object_detection.yml convert > $(DOCKER_COMPOSE)
+
+run_pytorch_object_detection: gen_pytorch_object_detection
```

Comment:
may want to include `xhost +local:docker` here. Otherwise, it will throw an error.